### ISSUE TITLE
Transfer learning loss issue

### ISSUE THREAD

**Issue by DNJ-JINI** [Kamlesh364/Mask-RCNN-TF2.7.0-keras2.7.0#12]
Hi,
I am trying to do transfer learning on my dataset with single class and it is working perfectly with tensorflow 2.4. 
But with tensorflow 2.7 loss is very high and  even after 50 epochs the predictions are wrong some random lines are seen in the output. 
Training epoch screenshot is attached for reference.
![image](https://user-images.githubusercontent.com/67704244/180603169-374f1250-505a-4655-8c88-7567972b5f7d.png)

Also tried prediction using mrcnn_predict.py with coco model and the predicted output is wrong. screenshot for the same is attached.
![image](https://user-images.githubusercontent.com/67704244/180604228-1b7785d9-c2ba-4e37-a71d-4caf9dbe1739.png)

Any help on this appreciated.

**Comment by DNJ-JINI** [comment]
Also I cannot find any change in the code  tf2.4 (model.py and utils.py) and tf 2.7

**Comment by Kamlesh364** [comment]
> Hi, I am trying to do transfer learning on my dataset with single class and it is working perfectly with tensorflow 2.4. But with tensorflow 2.7 loss is very high and even after 50 epochs the predictions are wrong some random lines are seen in the output. Training epoch screenshot is attached for reference. ![image](https://user-images.githubusercontent.com/67704244/180603169-374f1250-505a-4655-8c88-7567972b5f7d.png)
> 
> Also tried prediction using mrcnn_predict.py with coco model and the predicted output is wrong. screenshot for the same is attached. ![image](https://user-images.githubusercontent.com/67704244/180604228-1b7785d9-c2ba-4e37-a71d-4caf9dbe1739.png)
> 
> Any help on this appreciated.

please try [this](https://github.com/Kamlesh364/Mask-RCNN-TF2.7.0-keras2.7.0/issues/1#issuecomment-1112327292).

**Comment by DNJ-JINI** [comment]
Thanks @Kamlesh.it works fine.
Have you ever noticed any prediction accuracy issues with Tensorflow2.4 and Tensorflow2.7?
For me ,with same data tensorflow2.4 give better predictions compared to 2.7 with same number of epochs.

**Comment by Kamlesh364** [comment]
There should not be any difference based on the tensorflow version, until and unless there are some changes in the fundamental definitions in the framework.
